### PR TITLE
Add uikit to experimental/examples/imageviewer

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
- Add miniatures for big images. It speeds up launch.
 - Move most of the Desktop and Android code to commonMain.
 - Add UIKit target. For now, bitmap filters don't work for UIKIt.

### CODE REVIEW

File: experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/model/Config.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package example.imageviewer.model
+
+const val BASE_URL = "https://raw.githubusercontent.com/JetBrains/compose-jb/master/artwork/imageviewerrepo"
```
Comment (dima-avdeev-jb) [inline experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/model/Config.kt:3]: For now, If you want to run sample, change the url to:
```
"https://raw.githubusercontent.com/JetBrains/compose-jb/dima-avdeev/add-uikit-to-imageviewer/artwork/imageviewerrepo"
```

File: experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/FullscreenImage.kt [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    }
+
+    Box(Modifier.fillMaxSize()) {
+        Column {
+            Toolbar(picture?.name ?: "", filtersState, localization, back)
+            if (imageWithFilter != null) {
+                val imageSize = IntSize(imageWithFilter.width, imageWithFilter.height)
+                val scalableState = remember(imageSize) { mutableStateOf(ScalableState(imageSize)) }
+                val visiblePartOfImage: IntRect = scalableState.value.visiblePart
+                Slider(
```
Comment (igordmn) [inline experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/FullscreenImage.kt:74]: The slider and Back/Forward colors don't fit into imageviewer theme:
image

Let's change them to something else (more grayish probably, not sure)

**Review by igordmn — APPROVED** [review]
I would change colors of the new components, the rest of the PR is good.

File: experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/FullscreenImage.kt (hunk 2 above)
Comment (dima-avdeev-jb) [reply experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/FullscreenImage.kt:74]: Done

image

**Pull request merged**